core/plugins/plugins.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include <unordered_map>
+#include "NvInferPlugin.h"
+#include "NvInferPluginUtils.h"
+#include "core/util/prelude.h"
+
+namespace trtorch {
+namespace core {
+namespace plugins {
+namespace impl {
+// Helper class which registers a plugin in trtorch namespace
+template <typename T>
+class PluginRegistrar {
+ public:
+  PluginRegistrar() {
+    getPluginRegistry()->registerCreator(instance, "trtorch");

```

Comment:
Add a LOG_DEBUG Saying what Plugin is being registered in the TensorRT Plugin registry

Reply:
Modified this to include more debug information on what plugins are being registered.